### PR TITLE
Sort and update tags

### PR DESCRIPTION
**Pull request by Drsheppard01**

### CODE REVIEW

**Comment by loicreynier** [comment]
Thank you for this huge (tedious) work! The linter detected a few errors, could you correct them? I'll try to take a look at it in the coming days before merging.

**Review by loicreynier — APPROVED** [review]

**Comment by loicreynier** [comment]
Closes #226, closes #227.

**Comment by loicreynier** [comment]
@Drsheppard01 if you agree with my changes, we can merge it.

**Comment by Drsheppard01** [comment]
I make some changes, now it looks better

File: README.md [hunk 1]
```diff
@@ -404,7 +404,7 @@ this list aims to be broader and include apps from various other ecosystems in v
 - [Coffee](https://nick92.github.io/coffee) - News and weather reader for sources provided by News API and DarkSky `#vala` `#gtk3`.
 - [Feeds](https://gfeeds.gabmus.org) - News reader for the GNOME desktop `#python` `#gtk4` `#libadwaita` `#gnome`.
 - [Liferea](https://lzone.de/liferea) - News reader with a GUI similar to desktop mail client and with an embedded web browser `#c` `#gtk3`.
-- [NewsFlash](https://gitlab.com/news-flash/news_flash_gtk) - News reader designed to complement an already existing web-based RSS reader account `#rust` `#gtk4` `#libadwaita`.
+- [NewsFlash](https://apps.gnome.org/NewsFlash/) - News reader designed to complement an already existing web-based RSS reader account `#rust` `#gtk4` `#libadwaita` `#gnome`.
```
Comment (loicreynier) [inline README.md]:
```suggestion
- [NewsFlash](https://apps.gnome.org/NewsFlash) - News reader designed to complement an already existing web-based RSS reader account `#rust` `#gtk4` `#libadwaita` `#gnome`.
```

File: README.md [hunk 2]
```diff
@@ -576,14 +576,14 @@ Clients for commercial social platforms that had their API access cut off in a w
...
+- [Exercise Timer](https://github.com/mfep/exercise-timer) - Interval training timer `#rust` `#gtk4` `#libadwaita` `#relm4`.
 - [Flowtime](https://github.com/Diego-Ivan/Flowtime) - Pomodoro timer with statistics `#vala` `#gtk4` `#libadwaita`.
 - [Furtherance](https://github.com/lakoliu/Furtherance) - Cross-platform time tracker `#rust` `#gtk4` `#libadwaita`.
 - [hamster-gtk](https://github.com/projecthamster/hamster-gtk) - Time tracker provided by `hamster-lib` `#python` `#gtk3`.
 - [Hourglass](https://github.com/sgpthomas/hourglass) - Simple time keeping application for elementaryOS `#vala` `#gtk4` `#granite`.
 - [Khronos](https://apps.gnome.org/Khronos) - Task time logger `#vala` `#gtk4` `#libadwaita` `#gnome`.
 - [Retro](https://github.com/sonnyp/Retro) - Customizable digital clock `#gjs` `#gtk4` `#libadwaita`.
-- [Solanum](https://gitlab.gnome.org/World/Solanum) - Pomodoro timer for the GNOME desktop `#rust` `#gtk4` `#libadwaita` `#gnome`.
+- [Solanum](https://apps.gnome.org/Solanum/) - Pomodoro timer for the GNOME desktop `#rust` `#gtk4` `#libadwaita` `#gnome`.
```
Comment (loicreynier) [inline README.md]:
```suggestion
- [Solanum](https://apps.gnome.org/Solanum) - Pomodoro timer for the GNOME desktop `#rust` `#gtk4` `#libadwaita` `#gnome`.
```

File: README.md [hunk 3]
```diff
@@ -734,7 +734,7 @@ Clients for commercial social platforms that had their API access cut off in a w
 
 - [Black Fennec](https://gitlab.ost.ch/blackfennec/blackfennec) - Visual semi-structured data (JSON) editor `#python` `#gtk4` `#libadwaita`.
 - [KonbuCase](https://github.com/ryonakano/konbucase) - Case converting application `#vala` `#gtk4` `#libadwaita`.
-- [Wildcard](https://gitlab.gnome.org/World/Wildcard) - Regex tester `#rust` `#gtk4` `#libadwaita`.
+- [Wildcard](https://gitlab.gnome.org/World/Wildcard) - Regex tester `#rust` `#gtk4` `#libadwaita` `#gnome`.
```
Comment (loicreynier) [inline README.md:737]:
```suggestion
- [Wildcard](https://gitlab.gnome.org/World/Wildcard) - Regex tester `#rust` `#gtk4` `#libadwaita`.
```

Projects from `gitlab.gnome.org/World` are not part of the core GNOME applications.

**Review by loicreynier — CHANGES_REQUESTED** [review]

**Comment by Drsheppard01** [comment]
In continuation with the `#gnome` tag, would it be correct to also remove it for gedit and morphosis, because, they are also not gnome core apps, I also put the gnome tag next to the gnome circle apps, but not for gnome incubator apps

**Comment by loicreynier** [comment]
Yes indeed we should definitely remove the `#gnome` tag for Gedit and Morphosis, as they are neither part of the core apps nor part of GNOME Circle.

**Comment by loicreynier** [comment]
Closes #220.

**Comment by loicreynier** [comment]
Thanks @Drsheppard01, now merged.

**Pull request merged**